pytorch_lightning/accelerators/accelerator.py

```diff
@@ -36,11 +36,12 @@
 
 class Accelerator(object):
 
-    def __init__(self, trainer=None, cluster_environment=None):
+    def __init__(self, trainer=None, cluster_environment=None, ddp_plugin=None):
```

Comment:
revisiting this for https://github.com/PyTorchLightning/pytorch-lightning/issues/4349

Could we read `ddp_plugin` directly off `trainer.plugn_connector`? otherwise as we add more plugin types, this constructor will be huge